guides/create-graphql-mutation.md

```diff
@@ -0,0 +1,234 @@
+## At a glance
+
+You're working on an implementation for Leader of the Pack, an outdoor equipment retailer. In the GraphQL query guide we described data we added that contains whether we can ship a category of products to a particular zip. But now we need a way to update that information. For this we will create a GraphQL mutation
+
+## What you need 
+
+You need to understand what key you will use to query the data, and what the payload of the data is.
```

Comment:
We don't have it yet, but are the going to be inlinks into our GraphQL documentation or out to 3rd party GraphQL docs if they need more info on these two items?